album/homestuck-vol-9.yaml

```diff
@@ -262,6 +262,7 @@ Cover Artists:
 Art Tags:
 - Meenah
 Referenced Tracks:
+- Condescescion
```

Comment:
[Review in Discord.](https://discord.com/channels/749042497610842152/749067042250031214/1189296575428694026)